README.md

```diff
@@ -34,8 +38,8 @@ kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-ad
 ```
 This is important for the DCA and Kubernetes State metrics (explained later)
 
-### Cluster Exists already
-Store the Datadog API key in a kubernetes secret so its not directly in the deployment code
+### Cluster Exists already (@z: What does this title mean? It's not clear to me)
```

Comment:
I believe the meaning here is that a GKE cluster has already been provisioned, so you need not run the above `create` command. But perhaps at the beginning of the `GKE Start up` section we should mentioned that if you already have a cluster to skip ahead to this section (this section still applies after you've created a cluster, so perhaps it should be titled `Cluster Exists` instead)